src/cron_tasks/shared.py

```diff
@@ -41,7 +30,9 @@ async def get_model_inference_rows_df(
     )
     inference_rows_processed = [x.processed for x in inference_rows_in_db]
     inference_rows_nonprocessed = [x.nonprocessed for x in inference_rows_in_db]
-    inference_rows_actual = [x.actual for x in inference_rows_in_db]
+    inference_rows_actual = [
+        x.actual for x in inference_rows_in_db if x.actual is not None
+    ]
```

Comment:
@NickNtamp Should we assume that all inferences either have or don't have an actual? I mean they're no mix of inferences with an actual and inferences with no actual.

Reply:
@sinnec I believe that this is totally on client. I mean, I believe that we have to cover all the possible combinations (an inference set with all the actuals, an inference set with no actuals, an inference set with some actuals)

Reply:
Will open this as an issue for next iteration!